pdm/meta/action.yml

```diff
@@ -0,0 +1,100 @@
+
+name: Extract `pdm` metadata
+description: Extract required metadata from `pdm`
+
+inputs:
+  python-version:
+    description: Python version used by `pdm`
+    required: false
+    default: "3.11"
+    
+
+outputs:
+  has_tests:
+    description: Wether the project has tests exposed through the `test` command
+    value: ${{ steps.commands.outputs.tests }}
```

Comment:
I'm unsure we should allow a project not to have tests tbh.

Reply:
It's just introspection, you are not force to use it. It was done to be able to provide later a reusable generic workflow running only steps for which there is a command.

Reply:
Right :)